CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.9.0 - 2022-11-16
+
+### Added
+
+- Create Function button in Workspace view [#3430](https://github.com/microsoft/vscode-azurefunctions/pull/3430)
+- Warning when a stack has an upcoming end of life [#3382](https://github.com/microsoft/vscode-azurefunctions/pull/3382)
+- Updated templates to enable dynamic concurrency [#3365](https://github.com/microsoft/vscode-azurefunctions/pull/3365)
```

Comment:
I think it's worth listing which templates utilize dynamic concurrency so people don't assume it's enabled for everything.